src/PackageDetailsViewExtension/PackageDetailsViewExtension.csproj

```diff
@@ -1,82 +1,82 @@
 <Project Sdk="Microsoft.NET.Sdk">
-    <PropertyGroup>
-        <UILib>true</UILib>
-    </PropertyGroup>
+	<PropertyGroup>
+		<UILib>true</UILib>
+	</PropertyGroup>
   <ImportGroup Label="PropertySheets">
-    <Import Project="$(SolutionDir)Config\CS_SDK.props" />
+	<Import Project="$(SolutionDir)Config\CS_SDK.props" />
   </ImportGroup>
   <PropertyGroup>
-    <ProjectGuid>{12FB4DA4-7E9C-4C2E-91CD-1D0B523A9535}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Dynamo.PackageDetails</RootNamespace>
-    <AssemblyName>PackageDetailsViewExtension</AssemblyName>
-    <EnableDefaultPageItems>false</EnableDefaultPageItems>
+	<ProjectGuid>{12FB4DA4-7E9C-4C2E-91CD-1D0B523A9535}</ProjectGuid>
+	<OutputType>Library</OutputType>
+	<AppDesignerFolder>Properties</AppDesignerFolder>
+	<RootNamespace>Dynamo.PackageDetails</RootNamespace>
+	<AssemblyName>PackageDetailsViewExtension</AssemblyName>
+	<EnableDefaultPageItems>false</EnableDefaultPageItems>
   </PropertyGroup>
   <ItemGroup>
 	<PackageReference Include="Greg" Version="3.0.0.3175" />
 	<PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
-	<PackageReference Include="RestSharp" Version="108.0.1" />
+	<PackageReference Include="RestSharp" Version="109.0.1" />
```

Comment:
we cannot move to RestSharp 109 - Revit is stuck on 108.0.1

Reply:
Oh darn, because the restsharp 108 version API has a compatibility issue with the forge API. I will look into assembly isloation

Reply:
You could follow the same example as the ironpython extension https://github.com/DynamoDS/DSIronPython/blob/53e979c1f06f745e7fb43594bd7f4f525ed20f70/IronPythonExtension/IronPythonExtension.cs#L65

Reply:
That would mean to have a separate (separate from the extension assembly) assembly that brings in the reatsharp dependency and a dedicated subfolder for that assembly.